examples/guided_onboarding_example/public/components/app.tsx

```diff
@@ -30,62 +30,64 @@ export const GuidedOnboardingExampleApp = (props: GuidedOnboardingExampleAppDeps
   const { notifications, guidedOnboarding, history } = props;
 
   return (
-    <I18nProvider>
-      <EuiPageTemplate restrictWidth={true} panelled={true}>
-        <EuiPageTemplate.Header
-          pageTitle={
-            <FormattedMessage
-              id="guidedOnboardingExample.title"
-              defaultMessage="Guided onboarding examples"
-            />
-          }
-        />
-        {guidedOnboarding?.guidedOnboardingApi?.isEnabled ? (
-          <EuiPageTemplate.Section>
-            <Router history={history}>
-              <Routes>
-                <Route exact path="/">
-                  <Main notifications={notifications} guidedOnboarding={guidedOnboarding} />
-                </Route>
-                <Route exact path="/stepOne">
-                  <StepOne guidedOnboarding={guidedOnboarding} />
-                </Route>
-                <Route exact path="/stepTwo">
-                  <StepTwo />
-                </Route>
-                <Route exact path="/stepThree">
-                  <StepThree guidedOnboarding={guidedOnboarding} />
-                </Route>
-                <Route path="/stepFour/:indexName?">
-                  <StepFour guidedOnboarding={guidedOnboarding} />
-                </Route>
-              </Routes>
-            </Router>
-          </EuiPageTemplate.Section>
-        ) : (
-          <EuiPageTemplate.EmptyPrompt
-            iconType="error"
-            color="danger"
-            title={
-              <h2>
-                <FormattedMessage
-                  id="guidedOnboardingExample.errorTitle"
-                  defaultMessage="Guided onboarding is disabled"
-                />
-              </h2>
-            }
-            body={
-              <p>
-                <FormattedMessage
-                  id="guidedOnboardingExample.errorDescription"
-                  defaultMessage="Make sure your Kibana instance runs on Cloud and/or
-                  your user has access to Setup guides feature."
-                />
-              </p>
+    <EuiProvider colorMode="light">
```

Comment:
May I suggest using `KibanaRenderContextProvider` instead of `EuiProvider`? https://github.com/elastic/kibana/tree/d601e23c40be2b83c75be23c921ac42a39e034dd/packages/react/kibana_context/render

Doing so allow support dark mode to be supported, and remove the need to nest `I18nProvider` in the render tree.

Reply:
I was being lazy and figured we wouldn't care much about that app in dark mode... Thanks for mentioning this, you can find the fix in [this commit](https://github.com/elastic/kibana/pull/199070/commits/e6de3928dda9ccb155b499e1ca5a34f547da1854)